<launch>
  <arg name="device" default="can0"/>
  <arg name="steering_offset_deg" default="0.0"/>

  <node pkg="ymc" name="g30esli_interface" type="g30esli_interface" output="screen">
    <param name="device" value="$(arg device)"/>
    <param name="steering_offset_deg" value="$(arg steering_offset_deg)"/>
  </node>

</launch>
